common/modules/gcp-project/main.tf

```diff
@@ -33,9 +46,11 @@ resource "google_project" "project" {
 resource "google_project_services" "project" {
   project = "${google_project.project.project_id}"
   services = [
+    "oslogin.googleapis.com",
     "compute.googleapis.com",
     "logging.googleapis.com",
     "monitoring.googleapis.com",
+    "appengine.googleapis.com",
```

Comment:
This seems like a pretty weird API to need, although I see Stepan is having a problem related to appengine in [this comment](https://github.com/gpii-ops/gpii-infra/pull/60#issuecomment-413242122).

Do we have any idea what is pulling in a dependency on GAE?

Reply:
I ended up with quite a long list of APIs, which in my experience were required once you want GKE cluster, but `appengine.googleapis.com` wasn't one of those. 

I'm wondering if this will eventually start fighting with each other once the GKE cluster is created - code here removing APIs, code for GKE enabling them again... - but I didn't get that far yet.

Can't figure out what is causing my error - if I try to import that very same GCP project as TF resource somewhere else, everything works fine, but whenever I run our code I end up with the very same error. I guess I'll try to disable the appengine API - might be a good hint :).

Just fyi the APIs I needed for working GKE were:
```
    "compute.googleapis.com",
    "container.googleapis.com",
    "cloudbilling.googleapis.com",
    "cloudresourcemanager.googleapis.com",
    "dns.googleapis.com",
    "iam.googleapis.com",
    "serviceusage.googleapis.com",
    "storage-api.googleapis.com",
    "oslogin.googleapis.com",
    "bigquery-json.googleapis.com",
    "containerregistry.googleapis.com",
    "pubsub.googleapis.com",
    "deploymentmanager.googleapis.com",
    "replicapool.googleapis.com",
    "replicapoolupdater.googleapis.com",
    "resourceviews.googleapis.com",
```